<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaConfigEditor</class>
 <widget class="QDialog" name="MetaConfigEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="layoutGeneral">
       <item row="0" column="0">
        <widget class="QLabel" name="projectLabel">
         <property name="text">
          <string>Project:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="projectLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="titleLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTextEdit" name="descriptionTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
